### DISCUSSION TITLE
How to silently remove a software (PDF Creator.EXE not .MSI) set up on many PC ?

### DISCUSSION THREAD

**Discussion by hsychat** [Klocman/Bulk-Crap-Uninstaller#439] [Q&A]
Hello everyone !

I have silently deployed PDF Creator.exe by OCS Inventory on many PC.

Now I want to remove it from all the PC but the thing is as the soft is not on .MSI I can't manage to uninstall it.

I would like to create or export a rule or a config which allow me to launch it (by OCS or GPO)

Moreover, there are not any "unins000.exe" that i can call out from the folder (only on the PDFCreator Free version).

I spent so many hours on it... please if you have any idea :)

Best Regards,

Hans.

**Comment by Klocman** [comment]
See how the app looks like in BCU, it can give you a quiet uninstall string you can use as it is (if the uninstaller type supports it).
